Lesson8/src/com/handlers/HandlerLesson2.java

```diff
@@ -0,0 +1,25 @@
+package com.handlers;
+
+import com.TextFormatter;
+import com.tms.io.output.TmsFileWriter;
+import com.tms.io.output.TmsWriter;
+
+import java.io.IOException;
+
+public class HandlerLesson2 implements MyHandler {
+    final String OUTPUT_PATH;
+
+    public HandlerLesson2(String filePath) {
+        OUTPUT_PATH = filePath;
+    }
+
+    public void handleSomeString(String string) throws IOException {
+        String[] formattedData = TextFormatter.getSentences(string);
+        TmsWriter anotherWriter = new TmsFileWriter(OUTPUT_PATH);
+        for (String sentence: formattedData){
+            if (TextFormatter.checkPalindrome(sentence)){
```

Comment:
you also need to check the number of words here

Reply:
add check number in sout here

Reply:
this is not a check)

You just printing a number of words in console. But if number value from 3 to 5 - you need to append sentence to the writer

Reply:
reread the task, missed that part of the mind ((